next.config.ts

```
@@ -1,7 +1,38 @@
 };
 
-export default nextConfig;
+export default withSentryConfig(nextConfig, {
+// For all available options, see:
+// https://www.npmjs.com/package/@sentry/webpack-plugin#options
+
+org: "discovery-labs",
+project: "app-discoverycentre",
+
+// Only print logs for uploading source maps in CI
+silent: !process.env.CI,
```

ESLint

[error] Expected indentation of 2 spaces but found 0.

(style/indent)